frontend/server/src/Controllers/Course.php

```diff
@@ -882,7 +887,11 @@ public static function apiCreateAssignment(\OmegaUp\Request $r): array {
                 'assignment_type' => $r['assignment_type'],
                 'start_time' => $r['start_time'],
                 'finish_time' => $r['finish_time'],
-                'order' => intval($r['order']),
+                'order' => !empty($r['order'])
+                  ? intval($r['order'])
+                : \OmegaUp\DAO\Assignments::getNextPositionOrder(
+                    $course->course_id
+                ),
```

Comment:
se puede cambiar L860-L862 por

```
$order = $r->ensureOptionalInt('order');
```

para que esto pueda ser

```suggestion
                'order' => $order ?? \OmegaUp\DAO\Assignments::getNextPositionOrder(
                    $course->course_id
                ),
```

?

Reply:
Listo. Había olvidado que ya tenemos esas funciones para asegurar que es número.